server/sonar-server/src/main/java/org/sonar/server/ws/ServletRequest.java

```diff
@@ -67,4 +67,14 @@ protected InputStream readInputStreamParam(String key) {
     }
     return null;
   }
+
+  @Override
+  public String toString() {
+    StringBuffer url = source.getRequestURL();
+    String query = source.getQueryString();
```

Comment:
This will not work with POST requests and post-data parameters (but then POST requests can contain pretty big parameters).